lib/pages/community/community_about_tab.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+
+import '../../l10n/l10n.dart';
+import '../../stores/accounts_store.dart';
+import '../../util/extensions/spaced.dart';
+import '../../util/goto.dart';
+import '../../util/intl.dart';
+import '../../util/observer_consumers.dart';
+import '../../widgets/bottom_safe.dart';
+import '../../widgets/markdown_text.dart';
+import '../../widgets/pull_to_refresh.dart';
+import '../../widgets/user_tile.dart';
+import '../modlog_page.dart';
+import 'community_store.dart';
+
+class CommmunityAboutTab extends StatelessWidget {
+  const CommmunityAboutTab({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ObserverBuilder<CommunityStore>(builder: (context, store) {
+      final community = store.communityView!;
```

Comment:
You can pass it through the constructor if you need it to not be null

In fact same could be done with other widgets instead of returning SizedBox, that way you guarantee you are in the correct state